services/wallet/transfer/commands_sequential.go

```diff
@@ -1318,3 +1305,42 @@ func nextRange(maxRangeSize int, prevFrom, zeroBlockNumber *big.Int) (*big.Int,
 
 	return from, to
 }
+
+func accountLimiterTag(account common.Address) string {
+	return transferHistoryTag + "_" + account.String()
+}
+
+func createChainClientWithLimiter(client chain.ClientInterface, account common.Address, limiter chain.RequestLimiter) (chain.ClientInterface, error) {
+	// Each account has its own limit and a global limit for all accounts
+	accountTag := accountLimiterTag(account)
+	chainClient := chain.ClientWithTag(client, accountTag, transferHistoryTag)
+
+	// Check if limit is already reached, then skip the comamnd
+	if allow, err := limiter.Allow(accountTag); !allow {
```

Comment:
Do we disallow clients with limiter creation if the limit for an account is reached, but for history isn't?